engine/src/main/resources/sql/entandoServ_data_production.sql

```diff
@@ -40,6 +40,7 @@ INSERT INTO authuserprofiles (username, profiletype, profilexml, publicprofile)
 <profile id="admin" typecode="PFL" typedescr="Default user profile"><descr /><groups /><categories /><attributes><attribute name="fullname" attributetype="Monotext" /><attribute name="email" attributetype="Monotext" /></attributes></profile>
 ', 0);
 
+INSERT INTO api_oauth_consumers (consumerkey, consumersecret, name, description, callbackurl, scope, authorizedgranttypes, expirationdate, issueddate) VALUES ('swagger', 'swaggerswagger', 'Swagger', 'Swagger', '/api/webjars/springfox-swagger-ui/oauth2-redirect.html', 'global', 'authorization_code', '2028-10-10 00:00:00', '2008-10-10 00:00:00');
```

Comment:
In this part it would be better to add a complete URL including APP_BASE_URL. I was considering adding a SpringInitializer to add the value in the DB. I'm open to suggestions about smarter ways of doing this.

Reply:
@eugeniosant does this fit with the direction you're taking on the spring security update?

Reply:
if you want to fix the "relative" redirect_url after the initialization you can extend the init method of ConsumerManager to update the value of this url with the "base_url" given by the URLManager.
This update can be done only for whagger consumer (and for this reason this key could be defined in the system constants).